client/signup/steps/design-type-with-store/index.jsx

```diff
@@ -121,11 +118,11 @@ export default React.createClass( {
 			default:
 				return <PressableStoreStep
 							{ ... this.props }
-							ref={ this.onPressableStoreStepRef }
+							ref={ ( ref ) => this.pressableStore = ref }
```

Comment:
we should be careful here when sending an in-place-defined closure for `ref` because it will create a new function on every call to `render()` and as a result, re-render the thing even if nothing else has changed. you can see this fiddle and click on the buttons to see the calls to `ref`

http://jsbin.com/zazowerugo/edit?js,console,output